cypress/page/upload.page.ts

```diff
@@ -0,0 +1,29 @@
+class UploadFilePage {
+  private uploadFilePageURL: string;
+  private file: string;
+  private tittle: string;
+
+  constructor() {
+      this.uploadFilePageURL = 'https://the-internet.herokuapp.com/upload';
+      this.file = '#drag-drop-upload';
+      this.tittle = '#drag-drop-upload > .dz-preview > .dz-details > .dz-filename > span';
+  }
+
+  public visitUploadFilePage(): void {
+    cy.visit(this.uploadFilePageURL);
+  }
+
+  uploadFile(fileName: string): void {
+    cy.get(this.file).attachFile(fileName, { subjectType: 'drag-n-drop' });;
```

Comment:
not sure, but I think that you need to click on the upload button after the attachFile call, to effectively upload the file

Reply:
![image](https://user-images.githubusercontent.com/33491290/182648545-f6551b11-28ae-4230-9f92-1a36783e6452.png)
![image](https://user-images.githubusercontent.com/33491290/182649348-fd19b54e-7774-42dd-8fa4-e3724bd3bc21.png)
I think it is not necessary because it is a Drag and Drop Component

Reply:
yes, it is.

Look, the page that you should verify after upload is done is this one:
![image](https://user-images.githubusercontent.com/10209180/182659021-1f4f0476-7b4c-4b3a-9e22-4b90272dd5ea.png)

Reply:
is that I am using Drag and Drop, the process would not be different?
when I click on upload once my file is uploaded it sends me that error
![image](https://user-images.githubusercontent.com/33491290/182660858-f2c4b0e3-6e7c-4add-b166-10c53eec3055.png)
![image](https://user-images.githubusercontent.com/33491290/182660928-4d5bdbde-48e2-4add-b49d-7bb670d74d2f.png)
![image](https://user-images.githubusercontent.com/33491290/182660969-d9eaecc7-8de4-44ce-9f4f-06850187470e.png)

Reply:
Ok, got it